### ISSUE TITLE
Empy GPX for LAB

### ISSUE THREAD

**Issue by IDILIO49** [mirsch/lab2gpx#134]
When trying to get this Adventure Lab the gpx is almost empty without any point.
https://labs.geocaching.com/goto/5b1332c7-82a2-42a3-9b3d-a19ee61a3082
32.64406, -16.91391
Radius in km: 1
Max Caches:1

**Comment by SamHenkel** [comment]
The AdLab "Filho da Terra" has the status off**** (vulgo: archived)
You can test this by scnning the code with your smartphone. The AdLab will show in the App but couldn't be started.

**Comment by IDILIO49** [comment]
But in these cases normally it returns no GPX  at all saying no caches in that zone, but in this case it return an empty gpx...

**Comment by SamHenkel** [comment]
> but in this case it return an empty gpx...

No, its does contain data from AdventureLab **Vamos ao Parque**
That way it is **not** empty.

**Comment by SamHenkel** [comment]
The AdventureLab has the status "Off" and is not playable. Hence it's not downloadable as well
